code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -208,9 +208,11 @@
  * Energy Sword
  */
 /obj/item/melee/energy/sword
-	color
```

Comment:
don't remove this - it breaks code for some reason.

Reply:
if you want to remove this anyway, remove the corresponding
/obj/item/melee/energy/sword/color
in the cargo datums/wherever it is.

Reply:
if you want to remove it you have to remove /obj/item/melee/energy/sword/color in the code somewhere else so it compiles correctly.